project/Http4sPlugin.scala

```diff
@@ -33,7 +33,7 @@ object Http4sPlugin extends AutoPlugin {
   override def requires = MimaPlugin && ScalafmtPlugin
 
   val scala_213 = "2.13.0"
-  val scala_212 = "2.12.8"
+  val scala_212 = "2.12.10"
```

Comment:
Metals support for this isn't released yet. Do we have to bump this now?

Reply:
I thought it was, that PR was merged, I can revert. Was having some classpath issues thought this would be easy.

Reply:
I'm good to merge this after we revert this, so we don't break tooling (more than sbt-1.3.0 already broke mine! 😆)